runtime/vara/src/migrations.rs

```diff
@@ -16,7 +16,25 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-#[allow(unused)]
 use crate::*;
+use sp_runtime::traits::Get;
 
-pub type Migrations = (pallet_gear_program::migrations::MigrateToV3<Runtime>,);
+pub struct NominationPoolsMigrationV4OldPallet;
+impl Get<Perbill> for NominationPoolsMigrationV4OldPallet {
+    fn get() -> Perbill {
+        Perbill::from_percent(10)
```

Comment:
any comments on that?

Reply:
https://github.com/paritytech/polkadot-sdk/blob/0aeab381380107a6462fea240181f6e2a78f21b8/polkadot/runtime/westend/src/lib.rs#L1511C1-L1511C1